README.md

```diff
@@ -89,3 +89,21 @@ The tests may be failing because you have teared down postgres recently
 If you've got an old image hanging around it could cause failures. There are two things to try
 1. Firstly run `make pull` within tmp_ras_rm_docker_dev and then re-run `make test`
 2. Secondly if that doesn't work run `docker kill $(docker ps -qa) ; docker rm $(docker ps -qa) ; docker rmi $(docker images -qa)` to delete all images and then re-run `make test`
+#### Failing setup
+##### Resetting databases bug
+When running the acceptance tests, it may get stuck on `resetting databases`. This happens because a process gets into a deadlock and after a few minutes the `make` command should terminate. There is a way of terminating
+the process before it hits a timeout and quits.
+
+If you run the sql statement below. It should terminate the process in the deadlock and carry on running the acceptance tests. If it stops running
```

Comment:
🎨 `If you run the sql statement below, it` rather than 2 separate sentences IMO

Reply:
Sounds good 👍